person.rb

```diff
@@ -1,16 +1,19 @@
-require 'securerandom'
+require './nameable'
 
-class Person
+class Person < Nameable
```

Comment:
- Kindly person has mony rentals so Getters and Setters for `rentals` should be present with attr_accessor, as we intend to both access (get) as well as write (set) it and A method for adding rentals present. 👍 
please check the bellow code snippet 👇
```ruby
class Person
  attr_accessor :name, :age, :rentals
  .
  .

  def initialize(age, name = "Unknown", parent_permission = true)
    .
    .
    @rentals = []
  end

  def add_rental(book, date)
    Rental.new(date, book, self)
  end

  .
  .
end
```

Reply:
@binhussen Getters and setters for the rentals attribute have been added using attr_accessor. Additionally, the add_rental method has been included to allow a person to add rentals. Thank you for your suggestion.